pyaedt/modules/solutions.py

```diff
@@ -2418,81 +2378,219 @@ def _rotation_to_euler_angles(R):
+
+                obj_list = self._create_geometries(geometry_path)
+                if obj_list:
+                    items["model_info"] = obj_list
+                    self.model_info.append(obj_list)
+
+                if self._app.component_array:
+                    component_array = self._app.component_array[self._app.component_array_names[0]]
+                    items["component_objects"] = component_array.get_component_objects()
+                    items["cell_position"] = component_array.get_cell_position()
+                    items["array_dimension"] = [
+                        component_array.a_length,
+                        component_array.b_length,
+                        component_array.a_length / component_array.a_size,
+                        component_array.b_length / component_array.b_size,
+                    ]
+                    items["lattice_vector"] = component_array.lattice_vector()
+
+                with open(metadata_file_name, "w") as f:
+                    json.dump(items, f, indent=2)
+        elapsed_time = time.time() - time_before
+        self._app.logger.info("Exporting Embedded Element Patterns...Done: %s seconds", elapsed_time)
+        return path_dict
+
+    @pyaedt_function_handler()
+    def _create_geometries(self, export_path):
+        """Export geometry in OBJ format."""
+        self._app.logger.info("Exporting Geometry...")
+        model_pv = self._app.post.get_model_plotter_geometries(plot_air_objects=False)
+        obj_list = {}
+        for obj in model_pv.objects:
+            object_name = os.path.basename(obj.path)
+            name = os.path.splitext(object_name)[0]
+            original_path = os.path.dirname(obj.path)
+            new_path = os.path.join(os.path.abspath(export_path), object_name)
+
+            if not os.path.exists(new_path):
+                new_path = shutil.move(obj.path, export_path)
+            if os.path.exists(os.path.join(original_path, name + ".mtl")):
+                try:
+                    os.remove(os.path.join(original_path, name + ".mtl"))
+                except SystemExit:
+                    self.logger.warning("File can not be removed.")
+            obj_list[obj.name] = [
+                os.path.join(os.path.basename(export_path), object_name),
+                obj.color,
+                obj.opacity,
+                obj.units,
+            ]
+        return obj_list
+
+
 class UpdateBeamForm:
-    def __init__(self, ff):
+    """Update far field data. This class is used to interact with the far field theta and phi scan.
+
+    Parameters
+    ----------
+    ff : :class:`pyaedt.modules.solutions.FfdSolutionData`
+        Far field solution data instance.
+    farfield_quantity : str, optional
+        Quantity to plot. Default `"RealizedGain"`.
+    quantity_format : str, optional
+        Conversion data function.
+        Available functions are: `"dB10"`, `"dB20"`, "`abs"`, `"real"`, `"imag"`, `"norm"`, `"ang"`, `"and_deg"`.
```

Comment:
```suggestion
        Quantity to plot. The default is ``"RealizedGain"``.
    quantity_format : str, optional
        Conversion data function.
        Available functions are: ``"dB10"``, ``"dB20"``, ``"abs"``, ``"real"``, ``"imag"``, ``"norm"``, ``"ang"``, and ``"and_deg"``.
```
The ``farfield_quantity`` parameter is missing the list of possibilities contained in the earlier occurrences.